docs/source/repak/map.rst

```diff
@@ -0,0 +1,190 @@
+Map Files
```

Comment:
This file should, imo, not be a top-level file, but instead a subfile of a different header. Feel free to disagree tho

Reply:
What would you suggest? It's currently a top level file because I couldn't think of a nice way to structure the files in a way that works particularly well.
Potentially I could put it as the parent file of the various asset types, but then afaik I would lose the nice contents section. Not sure how much that matters to people though